### ISSUE TITLE
SwatchColorPicker with multiple rows has "read only" cells in Narrator

### ISSUE THREAD

**Issue by MBSilverstein** [microsoft/fluentui#15413]
### Environment Information

- **Package version(s)**: ^7.132.0
- **Browser and OS versions**: Edge Version 85.0.564.68 (Official build) (64-bit) / Windows 10

### Describe the issue:

In a SwatchColorPicker with multiple rows, Narrator states that each color cell is "read only" when they receive focus.

### Please provide a reproduction of the issue in a codepen:

On the documentation site, the example for "Simple swatch color picker with multiple rows and larger cells that updates its icon color and shows a preview color" demonstrates the issue:

https://fabricweb.z5.web.core.windows.net/oufr/7.145.0/#/examples/swatchcolorpicker

1. Load Narrator.
2. Select a color swatch inside the example with multiple rows.
3. Notice that Narrator says the option is "read only".

#### Actual behavior:

When reading the color name, Narrator adds "read only", which is confusing in this context.

#### Expected behavior:

Narrator should not claim the color swatch is "read only". The behavior should be the same as for the single row examples on the same page, with the additional information about the row and column.

Ideally, a fix for this would also fix issue #11480 by restructuring the control.

### Documentation describing expected behavior

https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a12f3779-bdf1-40c2-bbcc-0e314ea94858}

**Comment by paulgildea** [comment]
@MBSilverstein Thanks for the filing this issue!

**Comment by paulgildea** [comment]
@smhigley FYI
